.github/workflows/changelog.yml

```diff
@@ -9,15 +9,6 @@ jobs:
   createPullRequest:
     runs-on: ubuntu-latest
     steps:
-      - name: Install Go
-        uses: actions/setup-go@v2
```

Comment:
I think we still need the `setup-go` action here.

Reply:
@zlesnr got it, so this entire block stays 

```- name: Install Go
      uses: actions/setup-go@v2
      with:
        go-version: 1.14.x
```

Reply:
I think so yes.